analytical/tests/test_tag_piwik.py

```diff
@@ -67,3 +68,53 @@ def test_render_internal_ip(self):
         self.assertTrue(r.startswith(
             '<!-- Piwik disabled on internal IP address'), r)
         self.assertTrue(r.endswith('-->'), r)
+
+    def test_uservars(self):
+        context = Context({'piwik_vars': [(1, 'foo', 'foo_val'),
+                                          (2, 'bar', 'bar_val', 'page'),
+                                          (3, 'spam', 'spam_val', 'visit')]})
+        r = PiwikNode().render(context)
+        msg = 'Incorrect Piwik custom variable rendering. Expected:\n%s\nIn:\n%s'
+        for var_code in ['_paq.push(["setCustomVariable", 1, "foo", "foo_val", "page"]);',
+                         '_paq.push(["setCustomVariable", 2, "bar", "bar_val", "page"]);',
+                         '_paq.push(["setCustomVariable", 3, "spam", "spam_val", "visit"]);']:
+            self.assertIn(var_code, r, msg % (var_code, r))
+
+    @override_settings(ANALYTICAL_AUTO_IDENTIFY=True)
+    def test_default_usertrack(self):
+        context = Context({
+            'user': User(username='BDFL', first_name='Guido', last_name='van Rossum')
+        })
+        r = PiwikNode().render(context)
+        msg = 'Incorrect Piwik user tracking rendering. Expected:\n%s\nIn:\n%s'
```

Comment:
Maybe "Not found:" would be more appropriate than just "Expected:", because after all that's why the `in` assertion is failing. Or you incorporate "not found" in the error message beforehand. -- What do you think?

 (Same goes for all your tests below.)